src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -315,3 +315,26 @@
 
     :else
     constants/mainnet-chain-ids))
+
+(defn total-owned-collectible
+  [ownership]
+  (reduce (fn [acc item] (+ acc (js/parseInt (:balance item)))) 0 ownership))
+
+(defn total-owned-collectible-by-address
+  [ownership address]
+  (reduce (fn [acc item]
+            (if (= (:address item) address)
+              (+ acc (js/parseInt (:balance item)))
+              acc))
+          0
+          ownership))
+
+(defn calculate-owned-collectible
+  [ownership address]
+  (if address
+    (total-owned-collectible-by-address ownership address)
+    (total-owned-collectible ownership)))
```

Comment:
What is `ownership`? seems like a "new" entity.

How do we get that `ownership` and pass it to this function?

Since this function receives two parameters, the only possible way to execute the "else" branch of this `if`, is by passing `nil` or `false` to this function.
e.g.
`(calculate-owned-collectible "0x..." nil)`

IMO the API can be improved, probably we can instead pass a wallet account :thinking: 

wdyt? any specific reason about this implementation?

Reply:
Ownership is an object that we receive from backend and it exists on every collectible and it's named `ownership` there as well. Therefore, I didn't want to call it sth else.

I didn't want to pass a whole object just for an address. It was easier to go with a string.

Reply:
hmm, I think it's a poor naming tbh. It doesn't really convey it's meaning/ use case.
What is used for?

Reply:
Additionally, we can make this a 2-arity function so we can get rid of that `address` needing to be `false` or `nil` to execute the `else` branch

Reply:
Looks better @J-Son89 !

regarding to the 2 arity mentioned, it seems `address` is optional from the API perspective, so we can expose this function as a 1 parameter fn too:

```clojure
(defn total-owned-collectible
  ([ownership]
   (total-owned-collectible ownership false))
  ([ownership address]
   (reduce (fn [acc item]
             (if (or (not address) (= (:address item) address))
               (+ acc (js/parseInt (:balance item)))
               acc))
           0
           ownership)))
```

Reply:
I replaced my code with this new function. Thanks for sharing @J-Son89 @ulisesmac